common/src/main/java/com/microsoft/identity/common/internal/broker/BrokerData.kt

```diff
@@ -152,6 +152,7 @@ data class BrokerData(val packageName : String,
                 init {
                     add(prodMicrosoftAuthenticator)
                     add(prodCompanyPortal)
+                    add(prodLTW)
```

Comment:
For these two line of code, I am not sure if we missed them or it's not the right time to add them.

Reply:
Had a offline conversation with Dome, refrain the 2nd one for now (as it might affect the existing logic - just want to be sure)